src/pages/rest/tutorials/image/delete.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Delete an image 
+contributor_name: Goivvy LLC
+contributor_link: https://www.goivvy.com/
+description: We will learn how to delete an existing image 
+--- 
+
+# Delete an image
+
+We will delete an image with **id=695**. You can see image IDs by using a list API call - see a previous [step](https://developer.adobe.com/commerce/webapi/rest/tutorials/image/list/).
+
+**Endpoint:**
+
+```html
+DELETE http://domain.com/rest/default/V1/products/MJ03/media/695
+```
+
+The payload is empty.
```

Comment:
```suggestion
The payload must be empty.
```

Making this suggestion because I noticed if I left the previously used `body` in place, I received a `true` response, but the image was not deleted.